db/db_table_properties_test.cc

```diff
@@ -251,6 +251,96 @@ TEST_F(DBTablePropertiesTest, GetColumnFamilyNameProperty) {
   }
 }
 
+class DeletionTriggeredUniversalCompactionTestListener : public EventListener {
+ public:
+  void OnCompactionBegin(DB* , const CompactionJobInfo& ci) override {
+    ASSERT_EQ(ci.compaction_reason,
+              CompactionReason::kFilesMarkedForCompaction);
+  }
+
+  void OnCompactionCompleted(DB* , const CompactionJobInfo& ci) override {
+    ASSERT_EQ(ci.compaction_reason,
+              CompactionReason::kFilesMarkedForCompaction);
+  }
+};
+
+TEST_F(DBTablePropertiesTest, DeletionTriggeredUniversalCompactionMarking) {
```

Comment:
Can this be combined with the ```DeletionTriggeredCompactionMarking``` test and parameterized, with compaction style as the parameter?